mzLib/Readers/ExternalResults/ResultFiles/MsDataFileToResultFileAdapter.cs

```diff
@@ -29,12 +29,11 @@ public override MsDataScan GetOneBasedScanFromDynamicConnection(int oneBasedScan
         #region IResultFile Members
```

Comment:
Changes in this class enable MsDataFiles to be loaded through Alex's new FileReader methods